platform/mellanox/iproute2/Makefile

```diff
@@ -0,0 +1,49 @@
+SHELL = /bin/bash
+.ONESHELL:
+.SHELLFLAGS += -e
+
+MAIN_TARGET = iproute2-mlnx_$(IPROUTE2_MLNX_VERSION_FULL)_$(CONFIGURED_ARCH).deb
+IPROUTE2_DEB = iproute2_$(IPROUTE2_MLNX_VERSION_FULL)_$(CONFIGURED_ARCH).deb
+IPROUTE2_TMP := $(shell mktemp -u -d -t ipr.XXXXXXXXXX)
+
+$(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
+	# Remove any stale files
+	rm -rf iproute2-$(IPROUTE2_MLNX_VERSION)
+
+	wget -O iproute2_$(IPROUTE2_MLNX_VERSION).orig.tar.xz http://deb.debian.org/debian/pool/main/i/iproute2/iproute2_$(IPROUTE2_MLNX_VERSION).orig.tar.xz
+	wget -O iproute2_$(IPROUTE2_MLNX_VERSION_FULL).dsc http://deb.debian.org/debian/pool/main/i/iproute2/iproute2_$(IPROUTE2_MLNX_VERSION_FULL).dsc
+	wget -O iproute2_$(IPROUTE2_MLNX_VERSION_FULL).debian.tar.xz http://deb.debian.org/debian/pool/main/i/iproute2/iproute2_$(IPROUTE2_MLNX_VERSION_FULL).debian.tar.xz
+	dpkg-source -x iproute2_$(IPROUTE2_MLNX_VERSION_FULL).dsc
```

Comment:
Regarding this problem, https://salsa.debian.org/kernel-team/iproute2/-/blob/debian/5.10.0-4_bpo10+1/include/version.h, Did SDK agree to generate a patch based on debian variant https://github.com/liorghub/Spectrum-SDK-Drivers/blob/4.5.4150/iproute2/patches/5.10.0/0001-Added-sx_netdev-and-sx_bridge-support-to-iproute2.patch#L40

Reply:
No, they did not agree to that. I now added the missing patch to fix version identifier in version.h